test/integration/client-side-encryption/driver.test.ts

```diff
@@ -401,6 +413,176 @@ describe('Client Side Encryption Functional', function () {
       });
     }
   );
+
+  describe('CSOT on ClientEncryption', function () {
+    const metadata: MongoDBMetadataUI = {
+      requires: { clientSideEncryption: true, mongodb: '>=4.4' }
+    };
+    function makeBlockingFailFor(command: string, blockTimeMS: number) {
+      beforeEach(async function () {
+        const utilClient = this.configuration.newClient();
+        await utilClient.db('admin').command({
+          configureFailPoint: 'failCommand',
+          mode: { times: 2 },
+          data: {
+            failCommands: [command],
+            blockConnection: true,
+            blockTimeMS,
+            appName: 'clientEncryption'
+          }
+        } as FailPoint);
+        await utilClient.close();
```

Comment:
We now have helpers for this, if you're interested:

```suggestion
        await configureFailPoint(this.configuration, {
          configureFailPoint: 'failCommand',
          mode: { times: 2 },
          data: {
            failCommands: [command],
            blockConnection: true,
            blockTimeMS,
            appName: 'clientEncryption'
          }
        });
```

(same for clearfailpoint)